.env_SAMPLE

```diff
@@ -176,3 +176,10 @@ export HUD_API_ENDPOINT=http://localhost:8000/hud-api-replace/
 # export OIDC_OP_USER_ENDPOINT=http://localhost:8080/openid/userinfo
 # export OIDC_OP_JWKS_ENDPOINT=[Not used for test OIDC provider]
 # export OIDC_OP_ADMIN_ROLE=[Not supported by test OIDC provider]
+
+
+###############################
+# Deletion archival storage
+###############################
+
+# export ARCHIVE_FILESYSTEM=s3://<access key>:<secret key>@<bucket name>/<archive path>
```

Comment:
For local testing of the archival functionality developers might be more likely to use a local file (/tmp/json_archives) or an in-memory archive (mem:///archive) than S3. It'd be useful to document those options here as a comment.